docker-compose.newnomis.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.1'
+services:
+  new-nomis-ui:
+    image: mojdigitalstudio/new-nomis-ui:latest
+    networks:
+      - hmpps
+    depends_on:
+      - oauth-server
+      - elite2-api
+    container_name: new-nomis-ui
+    ports:
+      - '3001:3001'
```

Comment:
I think it would be much simpler if this was kept as 3000 and use of force changed, otherwise running nomis, PSH locally just gets annoying.  We have tried to keep nomis, PSH and omic as 3000, 3002, 3001 and others use unique ports above that...

Reply:
I'm guessing we don't have a list of unique ports which make choosing a unique one challenging... also sentence-planning, pathfinder and cat-tool don't seem to follow this convention and all run on port 3000 by default